README.md

```diff
@@ -1,5 +1,15 @@
 # ERC721R
 
+## How to Install
+
+```
+git clone https://github.com/exo-digital-labs/ERC721R.git
```

Comment:
i think we should remove this line. if you have this on your computer the steps are just `npm install`